newa/cli.py

```diff
@@ -94,14 +95,42 @@ def cmd_jira(ctx: CLIContext) -> None:
 
     for erratum_job in ctx.load_erratum_jobs('event-'):
         # read Jira issue configuration
-        # get list of matching actions
-
-        # for action in actions:
-        # create epic
-        # or create task
-        # or create sutask
-        # if subtask assoc. with recipes
-        # clone object with yaml
+        config = ErratumConfig.from_yaml_file(Path('component-config.yaml.sample'))
+
+        # TODO: record created/existing issues. Instead of `Any`, maybe something
+        # from the Jira library would be stored. Or just a Jira ticket ID.
+        known_issues: dict[str, Any] = {}
+
+        # Iterate over issue actions. Take one, if it's not possible to finish it,
+        # put it back at the end of the queue.
+        issue_actions = config.issues[:]
+
+        while issue_actions:
+            action = issue_actions.pop(0)
+
+            print(f'* Would create a {action.type.name} issue:')
+            print(f'     summary: {action.summary}')
+            print(f'     summary: {action.description}')
+
+            if action.id in known_issues:
+                raise Exception(f'Issue "{action.id}" is already created!')
```

Comment:
Does it make sense to raise an Exception here? In a respin, the epic would already exist, but `newa` would still need to create a new task (and subsequent subtasks) parented to it. Assuming that created/existing issues were queried just before (as mentioned in [line 100](https://github.com/RedHatQE/newa/pull/7/files#diff-fbfe615aa583117b7a57302f5da6ff45c1eda26a6f4b2406b68a09ffe1bd698aR100)), wouldn't it be more suitable to log the message and start another loop iteration?

Reply:
Good point, and it's definitely up to you to replace it with more fitting handling. As it is now, it exists only to do something and fail if there's a suspicious issue. I fully expect the whole loop to look completely different in a couple of weeks, once Jira-related code starts landing in :)